app-utils/kitty/spec

```diff
@@ -1,5 +1,5 @@
-VER=0.26.4
-SRCS="tbl::https://github.com/kovidgoyal/kitty/releases/download/v${VER}/kitty-${VER}.tar.xz"
-CHKSUMS="sha256::c09591ac7c908e6a0549f75cc061c6318f46261f9f5d3f3775120063c3a43bb7"
+VER=0.35.2
+SRCS="tbl::https://github.com/kovidgoyal/kitty/archive/refs/tags/v${VER}.tar.gz"
```

Comment:
Please use a Git tag:

```
git::commit=tags/v$VER::https://github.com/kovidgoyal/kitty
```

The use of platform-generated tarballs are no longer allowed, per the Styling Manual.

Reply:
> Please use a Git tag:
> 
> ```
> git::commit=tags/v$VER::https://github.com/kovidgoyal/kitty
> ```
> 
> The use of platform-generated tarballs are no longer allowed, per the Styling Manual.

DONT DO THAT

This original release tarball is upstream's CI geneated with docs or other no-platform independent resources. It isnt a github automatic generated source archive.